centreon/src/Core/Application/Common/UseCase/CliAbstractPresenter.php

```diff
@@ -0,0 +1,66 @@
+<?php
```

Comment:
Why is it in Core/Application/Common and not in Core/Common/Application ?

Reply:
To put it in the same place as the AbstractPresenter